src/Sentry.Unity/SentryInitialization.cs

```diff
@@ -26,15 +28,24 @@ public static class SentryInitialization
         internal static LogTimeDebounce LogTimeDebounce = new(TimeSpan.FromSeconds(1));
 
         internal static bool IsInit { get; private set; }
+        private static readonly JsonSerializerOptions _jsonOptions = new()
+        {
+            PropertyNamingPolicy = JsonNamingPolicy.CamelCase
+        };
+
+        private static string SentryOptionsJsonPath => $"Sentry/SentryOptionsJson";

```

Comment:
Why `Json` in the name of the file? We just assume `SentryOptions` is JSON and if it fails to deserialize we log an error out to the console so the user can manually inspect `SentryOptions` and delete it. Or raise an issue here explaining how that happened.

Reply:
Because it's `quick-dirty-example-of-json-as-config` :)

Reply:
I see. But since this isn't a draft PR I assume it was the form you were proposing we merge in